07-advanced-git/07-01-git-log/README.md

```diff
@@ -0,0 +1,116 @@
+
+And boom, this was what we wanted at the start. Now we can see that we have added the landing page and also changed the extension of the landing page.  
+
+But there is something weird here...  
+We can see that there is a commit tagged with origin/master, along with the commits of our current branch, frontend/landing. As we have established previously in chapter 6, all branches start off somewhere. In this example, the branch frontend/landing branches out from the origin/master branch. As such, git log makes a reference to this.
+
+Luckily, we don't have to imagine these relationships between branches and commits as there is a flag that provides us with that visualization.
+
+#### graph
+As mentioned earlier, to display the relationships between branches and commits, we can use the `--graph` flag. This flag shows a 'graph' of the commits, or the relationship between branches and commits.  
+Combining this with the `--oneline` flag we showed above, makes for a very clean and concise git log.  
+
+(Try this out yourself and see what happens!)
+
+```bash
+git log --oneline --graph
+```
+![Git log with oneline and graph](res/small_picture_3.png)
+
+## Looking at the big picture
+Now that you know how to look at commits tagged to your branch, you might also want to see how the entire repository might look like.  
+Luckily we have a flag to look at the log of the entire repository.  
+
+#### all
+The `--all` flag shows the `git log`, including all the commits from every branch you have in the repository.  
+We will also be combining this flag with all previous flags to get the best overview of a project. (In our opinion)
+```bash
+git log --oneline --graph --all
+```
+![Git log with oneline graph and all](res/big_picture_1.png)
+
+Now we can see that our frontend landing page feature has already been merged down to our local master branch. We can also see that after `git fetch` (To fetch any changes from GitHub), teammate B had already made more commits on his feature branch and the local `backend/start-server` branch is behind.  
+
+## Wrapping up
+To end off, the `git log` command actually has a lot more features and flags which are a bit more nuanced and situational. This chapter demonstrates the most commonly used features and flags which we hope will be usefully to you.  
+
+As mentioned above, below are additional resources:  
+[More features of git log](https://www.atlassian.com/git/tutorials/git-log)
+[git log documentation](https://git-scm.com/docs/git-log)
+
+Additional GUI applications that can help you visualise the git log:  
+[VSCode extension (Git Graph)](https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph)
+![Git graph looks like](res/git_graph_vscode.png)
+
+On Github:
+Actually there is a feature on github that also allows you to visualise the graph of your respository.  
+It is under `Insights` &rarr; `Network` in your respective github respository.  
+![Github graph](res/github_graph.png)
```

Comment:
```suggestion
## Wrapping up
To end off, the `git log` command actually has a lot more features and flags which are a bit more nuanced and situational. This chapter demonstrates the most commonly used features and flags which we hope will be useful to you.  

As mentioned above, below are additional resources.

### Additional Resources to read
- [More features of git log](https://www.atlassian.com/git/tutorials/git-log)
- [git log documentation](https://git-scm.com/docs/git-log)

### GUI applications to help you visualise the git log
- [VSCode extension (Git Graph)](https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph)
![Git graph looks like](res/git_graph_vscode.png)

#### Github:
Actually there is a feature on GitHub that also allows you to visualise the graph of your repository.  
It is under `Insights` &rarr; `Network` in your respective GitHub repository.  
![Github graph](res/github_graph.png)
```

Compile most of the last few changes into one.
Thought it would be clearer like this

Reply:
Yeap, this is definitely a lot neater and will show up when rendered into a PDF in the table of contents :+1: If these are the final changes, feel free to commit them :smile: